modules/ROOT/pages/sap/sap-connector-terminology.adoc

```diff
@@ -0,0 +1,59 @@
+= SAP Terminology
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following are commonly used SAP system acronyms:
+
+ABAP (Advanced Business Application Programming)::
+SAP proprietary programming language and part of the NetWeaver platform for building business applications.
+
+ALE (Application Link Enabling)::
+Sets up distributed applications and integrated installation of SAP systems. This involves business-driven message exchange using consistent data across loosely linked SAP apps. Apps are integrated through synchronous and asynchronous communication, rather than by use of a central database.
+
+BAPI (Business Application Programming Interface)::
+The function module that fulfills design criteria, such as:
+** Implement a method of a SAP business object
+** Maintain a static interface through different versions of the SAP system
+** Enable remotely
```

Comment:
This begs for an object: enabling what remotely? Or do we want "Remote enablement"?

Reply:
...or even Providing you the ability to...do these things?

Reply:
Yes, it's remote enablement. I'll figure out a way to reword.